### PR TITLE
Aligned names of DC files

### PR DESCRIPTION
**Pull request by tbazant**
- [ ] for Travis: https://github.com/SUSEdoc/susedoc.github.io/blob/master/config.xml
- [ ] for docserv.suse.de: ~/daps-autobuild_docserv.xml
- [ ] for doc.suse.com: config.d/ses.xml
- [ ] not sure whether you create a packaged version of the SES docs, the package may need an update too

### CODE REVIEW

**Review by asettle — APPROVED** [review]

**Comment by sknorr** [comment]
Looks good. Only thing I would consider changing back is DC-windows, because the product's/feature's name is literally "SES for Windows." Your decision though.

Also, I have not checked the xml:ids in books, I guess it might be beneficial if those were updated too.

Finally, please note that should the SES 7 documents already be in the hands of translators, you may have a bunch of manual-editing pain coming. If you have not submitted the SES 7 documents to translation, you are golden on that front.

If you merge, we will need updates in the following spots:
* for Travis: https://github.com/SUSEdoc/susedoc.github.io/blob/master/config.xml
* for docserv.suse.de: ~/daps-autobuild_docserv.xml
* for doc.suse.com: config.d/ses.xml
* not sure whether you create a packaged version of the SES docs, the package may need an update too

**Comment by tbazant** [comment]
@sknorr  see points below
> Looks good. Only thing I would consider changing back is DC-windows, because the product's/feature's name is literally "SES for Windows." Your decision though.
DONE
> 
> Also, I have not checked the xml:ids in books, I guess it might be beneficial if those were updated for consistency too.

this is going to be the next step, renaming books and book IDs. do oyu think it's better to do all these changes in one PR?
> 
> Finally, please note that should the SES 7 documents already be in the hands of translators, you may have a bunch of manual-editing pain coming. If you have not submitted the SES 7 documents to translation, you are golden on that front.

files have not yet been submitted to translators, we're golden

> 
> If you merge, we will need updates in the following spots:
> 
>     * for Travis: https://github.com/SUSEdoc/susedoc.github.io/blob/master/config.xml
> 
>     * for docserv.suse.de: ~/daps-autobuild_docserv.xml
> 
>     * for doc.suse.com: config.d/ses.xml
> 
>     * not sure whether you create a packaged version of the SES docs, the package may need an update too

**Comment by sknorr** [comment]
> @sknorr see points below
> 
> > Looks good. Only thing I would consider changing back is DC-windows, because the product's/feature's name is literally "SES for Windows." Your decision though.
> > DONE
> > Also, I have not checked the xml:ids in books, I guess it might be beneficial if those were updated for consistency too.
> 
> this is going to be the next step, renaming books and book IDs. do oyu think it's better to do all these changes in one PR?
I don't mind if it's a single PR or two PRs here, but from the point of view of publishing on doc.suse.com, I'd like this to be one change. Ideally, don't request a new SES 7 build when only one of the PRs is merged.


> files have not yet been submitted to translators, we're golden

:+1:

**Comment by asettle** [comment]
@tbazant I'd actually vote for DC-ses-windows to remain. @sknorr is right, that's the name of the product which differs to the "Deployment Guide" theme.

**Review by lproven — APPROVED** [review]

**Review by sknorr — APPROVED** [review]
The changes here look good.

(Do be aware that your CI is failing because the IDs in the Manpage book contain underscores. This may obscure additional issues.)

File: DC-rook [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ##
 ## Basics
 MAIN="MAIN.susestorage.xml"
-ROOTID=book-storage-rook
+ROOTID=rook-guide
```
Comment (asettle) [inline DC-rook:8]: This is the only one I consider odd. But I think it's for the best. It's not going to last long anyway 👀

**Review by asettle — APPROVED** [review]

**Comment by asettle** [comment]
> The changes here look good. Thanks!
> 
> (Do be aware that your CI is failing because the IDs in the Manpage book contain underscores. This may obscure additional issues.)

The creation of the mange pages was automated with @tomschr script - we didn't have an issue before?

**Pull request closed without merging**